tests/template_tests/test_autoescape_setting.py

```diff
@@ -0,0 +1,22 @@
+
+
+from django.template import EngineHandler
+from django.test import SimpleTestCase
+
+
+class AutoescapeSettingTest(SimpleTestCase):
+
+    """ Test for autoescape setting for DjangoTemplates, #25469 """
```

Comment:
No need to reference the ticket unless it's a complex issue that requires additional context of the ticket.